docs/source/general/finetuning.rst

```diff
@@ -133,11 +100,16 @@ freeze
 ------
 The freeze strategy keeps the backbone frozen throughout.
 
-.. code-block:: python
+.. testcode:: strategies
+
+    trainer.finetune(model, datamodule, strategy="freeze")
```

Comment:
if it is testcode is called what is the execution time. full training?

Reply:
Trains just one batch for one epoch (see setup block in line 58)